numpy/__init__.py

```diff
@@ -156,11 +159,27 @@
     from . import matrixlib as _mat
     from .matrixlib import *
 
-    # Make these accessible from numpy name-space
-    # but not imported in from numpy import *
-    # TODO[gh-6103]: Deprecate these
-    from builtins import bool, int, float, complex, object, str
-    from .compat import long, unicode
+    # Deprecations introduced in NumNy 1.18.0, 2019-11-11
```

Comment:
```suggestion
    # Deprecations introduced in NumPy 1.20.0, 2020-06-11
```
(or some other date, closer to now :))